site/dat/docs/Apiritif.md

```diff
@@ -174,6 +174,20 @@ We can compose following XPath expressions:
 
 Any of these expressions can be used to fetch the desired element, if these attributes are unique.
 
+__7. By Shadow Locator__
+
+Shadow Locator enables to work with Shadow DOM which is designed as a tool for building component-based 
+applications. For example Salesforce Lightning Component Library is based on Shadow DOM. 
+Shadow DOM is self-contained and thus simple locators will not return nodes inside it.
+
+Shadow Locator is composed by a sequence of css selectors, e.g.:
+
+```html
+shadow: c-basic, lightning-accordion-section, .slds-button
+```
```

Comment:
please also add this (taken from docs you provided)
```
The last CSS Selector refers to the target element in a shadow tree. 
The preceding CSS Selectors refer to the shadow hosts.
```

Reply:
Good point, I will add it to the docs.